spec/integration/media_convert_adapter_spec.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+require 'spec_helper'
+require 'aws-sdk-cloudwatchevents'
+require 'aws-sdk-cloudwatchlogs'
+require 'aws-sdk-mediaconvert'
+require 'aws-sdk-s3'
+require 'json'
+require 'active_encode/spec/shared_specs'
+require 'active_support/json'
+require 'active_support/time'
+
+def reconstitute_response(fixture_path)
+  HashWithIndifferentAccess.new(ActiveSupport::JSON.decode(File.read(File.join("spec/fixtures", fixture_path))))
+end
+
+describe ActiveEncode::EngineAdapters::MediaConvertAdapter do
+  around do |example|
+    ActiveSupport.parse_json_times = true
+    Time.zone = 'America/Chicago'
```

Comment:
Should these be reset after the example runs?  Probably not a big deal right now because the tests are passing.

Reply:
Probably safer to reset them. Will do.